bin/posydon-run-pipeline

```diff
@@ -434,9 +437,7 @@ def train_interpolators(i, path_to_csv_file, verbose=False):
                     "out_keys": out_keys,
                     "out_nan_keys": out_nan_keys,
                     "class_method": "kNN",
-                    "c_keys": [f'S{i}_{MODEL_NAME}_CO_interpolation_class', 
-                               f'S{i}_{MODEL_NAME}_CO_type',
-                               f'S{i}_{MODEL_NAME}_SN_type'],
+                    "c_keys": [f'S{i}_{MODEL_NAME}_CO_interpolation_class',],
```

Comment:
This reverts a change done in PR #325 :
@maxbriel , could you please comment, why this change need to be reverted?
@ZepeiX , could you please comment, why you needed this change, and whether it is still needed?

Reply:
This was to fix the inconsistency between CO_type and CO_interpolation_class. I think it's still needed. 
Perhaps remove the SN_type and CO_type at line 401 when training for mass transfer cases.

Reply:
@ZepeiX I don't understand how this can fix the inconsistency after conversations with @philipp-rajah, since the classifiers are independent of the interpolation class and of each other. The interpolator is conditional to `c_key` classes, but the classifiers (`c_keys`) are not. The code seems to indicate this too.
I think it's your other change of the valid classes that fixed it, since that does play a role in the classifier training.

I will train the interpolators with this branch for an additional check.

Reply:
@maxbriel @mkruckow 
Did you see NSs and BHs with wrong masses again with your test?  I did find training the CO_type again gives different results. 
Can you add it back and train the interpolators again and to see if the wrong masses are fixed?

Reply:
It looks like you are only going with try and error here.
@ZepeiX, please explain, what it does differently by adding this, it looks wrong to us to put it in there again.
@philipp-rajah, could you may have a look at it, too? All this discussion may relate to unclear differences between `c_keys` and `c_key`. Unfortunately, the docstrings only include `c_keys` but don't explain `c_key`.

Reply:
I would like to add another argument to at least including it in the first training:

`S2_MODEL_CO_type` and `S2_MODEL_SN_type` classifiers are **not** included in the secondary training loop.